### ISSUE TITLE
[A11yInspect] Fail: 1.3.1 A in /home/runner/work/test-app/test-app/src/App.tsx

### ISSUE THREAD

**Issue by sitaram-r** [sitaram-r/test-app#25]
## ❌ Accessibility Fail

**Rule:** 1.3.1 A
**File:** `/home/runner/work/test-app/test-app/src/App.tsx`
**Line:** 1, **Column:** 1

### Description
Missing table header (Major)

### Context
```html
<table border="1">
        <tbody><tr>
          <td>Name</td>
          <td>Age</td>
        </tr>
        <tr>
          <td>John</td>
          <td>25</td>
        </tr>
      </tbody></table>
```

### Selector
`html > body > div > audio > table`

---
*Generated by [A11yInspect](https://github.com/barrierbreak/vscode-scally)*
